src/display/gl/transform.h

```diff
@@ -52,6 +52,10 @@
 #include <math.h>
 #include <string.h>
 
+#ifndef M_PI
+# define M_PI 3.14159265358979323846
+#endif
```

Comment:
This is needed because mkxp-z uses C++14 while `std::numbers::pi` isn't a thing until C++20, right?

Reply:
We're already using the exact same macro in sprite.cpp. However, I ended up not using the function that was using that macro in this PR, so I just removed it.